sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs

```diff
@@ -372,7 +610,7 @@ public partial class ImageVersionData : Azure.ResourceManager.Models.ResourceDat
         public bool? ExcludeFromLatest { get { throw null; } }
         public string NamePropertiesName { get { throw null; } }
         public int? OSDiskImageSizeInGb { get { throw null; } }
```

Comment:
`Gb` -> `GB`, this should apply to all if there have others

Reply:
Updated